relational/table.py

```diff
@@ -364,9 +373,9 @@ def _crunch_number(prefix):
                       'residential_units_adu_2br',
                       'residential_units_adu_3br',
                       'residential_units_studio',
-                      'residential_units_1br',
-                      'residential_units_2br',
-                      'residential_units_3br',
+                      self.OUT_1BR,
+                      self.OUT_2BR,
+                      self.OUT_3BR,
```

Comment:
note: missing OUT_4BR

Reply:
doesn't exist in PPTS, adding a comment to this effect